doc/whats_new/v0.20.rst

```diff
@@ -376,6 +376,10 @@ Imputer
 - Deprecate :class:`preprocessing.Imputer` and move the corresponding module to
   :class:`impute.SimpleImputer`. :issue:`9726` by :user:`Kumar Ashutosh
   <thechargedneutron>`.
+- Future (and default) behavior is equivalent to ``axis=0``
+  (impute along columns). Row-wise imputation can be performed with
+  FunctionTransformer (e.g.,
+  ``FunctionTransformer(lambda X: Imputer().fit_transform(X.T).T)``).
```

Comment:
Please mention the deprecation of `axis` parameter here and in the deprecation message of `preprocessing.Imputer`

Reply:
OK! @qinhanmin2014